components/centraldashboard/public/components/resources/cloud-platform-data.js

```diff
@@ -0,0 +1,22 @@
+export const getGCPData = (project) => ({
```

Comment:
I think this is perfect for future contributors to add functions for other platforms. It might be worth adding a top-level module comment block explaining the purpose of the module as well as a function level doc comment here.